pkg/repair/repair.go

```diff
@@ -59,32 +62,80 @@ func NewBrokenPodReconciler(client client.Interface, filters *Filters, options *
 	}
 }
 
+func (bpr BrokenPodReconciler) ReconcilePod(pod v1.Pod) (err error) {
+	errors := []string{}
+
+	log.Debugf("Reconciling pod %s", pod.Name)
+
+	if bpr.Options.LabelPods {
+		newErr := bpr.labelBrokenPod(pod)
+		if newErr != nil {
+			errors = append(errors, newErr.Error())
+		}
+	}
+
+	if bpr.Options.DeletePods {
+		newErr := bpr.deleteBrokenPod(pod)
+		if newErr != nil {
+			errors = append(errors, newErr.Error())
+		}
+	}
+
+	//if bpr.Options.CreateEvents {
+	//	newErr := bpr.createEventForPod(pod)
+	//	if newErr != nil {
+	//		errors = append(errors, newErr)
+	//	}
+	//}
+
+	errStr := strings.Join(errors, ",")
```

Comment:
nit: ive seen `multierr` used before if you prefer and if its already imported here. no personal preference though